test/functional/wallet_multiwallet.py

```diff
@@ -68,6 +70,8 @@ def wallet_file(name):
         wallet_names = ['w1', 'w2', 'w3', 'w', 'sub/w5', os.path.join(self.options.tmpdir, 'extern/w6'), 'w7_symlink', 'w8', '']
         extra_args = ['-wallet={}'.format(n) for n in wallet_names]
         self.start_node(0, extra_args)
+        assert_equal(set(map(lambda w: w['name'], self.nodes[0].listwalletdir()['wallets'])), set(['', 'w3', 'w2', 'sub/w5', 'w7', 'w7', 'w1', 'w8', 'w']))
```

Comment:
This is wrong, `fs::relative` resolves symlinks but we allow symlinks to be specified as wallet names. I guess `listwalletdir` should list both `w7` and `w7_symlink` but in runtime only one could be loaded (actually it's the same wallet but with different names). @ryanofsky @laanwj should I fix this in #14531 or fix in a separate PR?